src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private static final Random random = new Random();
+
+    public static Ball getRandomBall() {
+        Color randomColor = ColorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(100) + 1;
+        return new Ball(randomColor, randomNumber);
+    }
```

Comment:
Avoid using static methods. Instead, create an instance method that can be called on an object of the Lottery class. This will make your code more flexible and object-oriented.